packages/~/app/layout/src/__snapshots__/main.test.tsx.snap

```diff
@@ -55,16 +55,12 @@ exports[`Main Layout 1`] = `
           href="/node_modules/animate.css/source/bouncing_entrances/bounceIn.css"
         />
 
-        <!--  -->
-
-        <link
-                rel="stylesheet"
-                href="/node_modules/@gouvfr/dsfr/dist/dsfr/dsfr.min.css"
-              />
-              <link
-                rel="stylesheet"
-                href="/node_modules/@gouvfr/dsfr/dist/utility/utility.min.css"
-              />
+        <style>
+              @import "/node_modules/@gouvfr/dsfr/dist/dsfr/dsfr.css"
+                layer(dsfr);
+              @import "/node_modules/@gouvfr/dsfr/dist/utility/utility.css"
+                layer(dsfr-utility);
+            </style>
```

Comment:
indentation problem, is it possible to control the indentation of a snapshot?

Reply:
dunno